### PR TITLE
Governance params and super tokens page

### PR DESCRIPTION
**Pull request by Mikkoun**
Tooltips are still being reviewed by Nuno.

### CODE REVIEW

File: src/redux/protocolContracts.ts [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  },
+  "avalanche-fuji": {
+    resolver: "0x141920741bC45b962B59c833cd849bA617F7ef38",
+    host: "0xf04F2C525819691ed9ABD3D2B7109E1633795e68",
+    CFAv1: "0xED74d30B8034152b0638CB03cc5c3c906dd1c482",
+    IDAv1: "0x997d745884F54a93E6662b055c5e6c09F688718b",
+    superTokenFactory: "0xA25dbEa94C5824892006b30a629213E7Bf238624",
+    superfluidLoaderv1: "0x777Be25F9fdcA87e8a0E06Ad4be93d65429FCb9f",
+  },
+} as NetworkContracts;
```
Comment (kasparkallas) [inline src/redux/protocolContracts.ts:117]: In places like this, I'd rather do: `const protocolContracts: NetworkContracts = { ...`

**Review by kasparkallas — APPROVED** [review]

**Comment by vmichalik** [comment]
Looks excellent overall, really cool and will bring in way more people to using the console!
Few ideas:

- hover states seem to be missing, especially for the top right links
- sort options for token table
- default sort on tokens by isListed, then by alphabetical
- tooltip on listed in super tokens table
- scanner links next to copy icons in protocol governance addresses

future observations:
- we'll need a better way of showing network names to select rather than the scrollable bar when we add the next few

**Comment by elvijsTDL** [comment]
The splitting lines between rows we have everywhere else would also be nice in the governance page
Screenshot 2022-04-19 at 12 37 20

Screenshot 2022-04-19 at 12 37 41

**Comment by vmichalik** [comment]
- also, patrician period is defined as 1/8 of the total period, not as 30min
this is generally not relevant, but it can be in certain cases

**Comment by elvijsTDL** [comment]
> * also, patrician period is defined as 1/8 of the total period, not as 30min
>   this is generally not relevant, but it can be in certain cases

For testnets its 12 minutes and 1 hour so that would be 1/5th

**Comment by Mikkoun** [comment]
- Super tokens can not be ordered by name and listed status.
- Added tooltip for listed field
- Some other minor UI fixes

I wouldn't add default sorting to super tokens table because we can only order by one field at a time and if we ordered by name then all the first tokens would be without name - basically it would not look good.

**Review by elvijsTDL — APPROVED** [review]

**Pull request merged**